crm_lead_product/report/crm_product_report.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (C) 2017-19 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from psycopg2.extensions import AsIs
+
+from odoo import fields, models, tools
+
+
+class ActivityReport(models.Model):
+    """CRM Lead Analysis"""
+
+    _name = "crm.product.report"
+    _auto = False
+    _description = "CRM Pipeline by Product Analysis"
```

Comment:
Why adding a new report while the product can be injected in existing ones?

Reply:
At first sight it seems that this report is based on the new crm.lead.line model, depending on what you want to query, it will be more optimal to use one or the other and not penalise the performance of the existing one if you do not want to query anything at line level.
But come on, let @AdriaGForgeFlow answer if she wants to.:stuck_out_tongue_winking_eye: